src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+// variables for header
```

Comment:
Explanations through comments should be avoided, it is better to think of variable names that will explain themselves.